test/server/listener_manager_impl_test.cc

```diff
@@ -130,7 +130,7 @@ per_connection_buffer_limit_bytes: 8192
   EXPECT_EQ(8192U, manager_->listeners().back().get().perConnectionBufferLimitBytes());
 }
 
-TEST_F(ListenerManagerImplWithRealFiltersTest, TlsTransportSocket) {
+TEST_F(ListenerManagerImplWithRealFiltersTest, DEPRECATED_FEATURE_TEST(TlsTransportSocket)) {
```

Comment:
test nit: I would fix this test, since it's really not that deprecated behavior you care about. Instead, I would have a specific TLS test that tests the old deprecated behavior and thus a DEPRECATED_FEATURE_TEST there? WDYT?

Reply:
Yeah. That makes. I will fix it. We already have a TLSContext test that I marked deprecated.